.freebsd.test.py

```diff
@@ -31,13 +31,17 @@
 
 def main():
     judgeenv.env['runtime'] = {}
-    judgeenv.env['extra_fs'] = {'PERL': ['/dev/dtrace/helper$'], 'RUBY2': ['/dev/dtrace/helper$']}
+    judgeenv.env['extra_fs'] = {
+        'PERL': {"exact_file": ["/dev/dtrace/helper"]},
+        'RUBY2': {"exact_file": ["/dev/dtrace/helper"]}
```

Comment:
This is not what was agreed to earlier. The YAML we actually agreed to was:

```suggestion
        'PERL': [{"exact_file": "/dev/dtrace/helper"}],
        'RUBY2': [{"exact_file": "/dev/dtrace/helper"}],
```

Reply:
Why would it be a list of dictionaries

Reply:
```yaml
extra_fs:
  PERL:
  - file: /dev/dtrace/helper
  - dir: /some/exact/directory
  - recurse: /some/directory/and/all/children
```

This is what we agreed to earlier. Clearly, `extra_fs.PERL` is a list of dictionaries, each with one key/value pair.